Trimble.Modus.Components/Controls/ListView/TMListView.xaml.cs

```diff
@@ -125,22 +125,22 @@ private void UpdateCellColor()
             {
                 if (SelectableItems.Contains(textCell.BindingContext))
                 {
-                    textCell.UpdateBackgroundColor(ResourcesDictionary.ColorsDictionary(ColorsConstants.BluePale));
```

Comment:
You should try the VisualStateManager for this one. The state should go from 'Normal' to 'Selected' here